src/pages/Chat.vue

```diff
@@ -8,7 +8,7 @@
     <q-scroll-area
       ref="chatScroll"
       class="q-px-md row"
-      :style="`height: calc(100% - ${offsetHeight()}px); min-height: calc(100%-${offsetHeight()}px); background: lightblue`"
+      :style="`height: calc(100% - ${offsetHeight()}px); min-height: calc(100%-${offsetHeight()}px); background: ${bodyBackground}`"
```

Comment:
Is it possible to actually create a CSS theme to avoid passing all this stuff around?

Reply:
definitely, I will see what I can do